src/nupic/algorithms/Connections.hpp

```diff
@@ -528,14 +533,9 @@ class Connections : public Serializable
   void unsubscribe(UInt32 token);
 
 protected:
-  /**
-   * Gets the synapse with the lowest permanence on the segment.
-   *
-   * @param segment Segment whose synapses to consider.
-   *
-   * @retval Synapse with the lowest permanence.
-   */
-  Synapse minPermanenceSynapse_(Segment segment) const;
+  void remove_presynapse_(UInt index,
```

Comment:
nit, use thisNamingConvention

Reply:
I think that this has already been renamed to method: `removeSynapseFromPresynapticMap_`